server.py

```diff
@@ -156,6 +162,10 @@ def shutdown():
     func()
     return 'Server is shutting down...', 200
 
+@app.route('/earliestDate', methods=['GET'])
+def get_earliest_date():
```

Comment:
nope hahaha i thought i was going to use that, but nope too complicated. :laughing: